chucks-pick3-ws/src/main/java/software/bytepushers/pick3/services/Pick3PlaysServiceImpl.java

```diff
@@ -13,18 +12,15 @@
 public class Pick3PlaysServiceImpl implements Pick3PlaysService {
 
     private final Pick3PredictionService pick3PredictionService;
-    private final Pick3PlaysConfig config;
 
-    public Pick3PlaysServiceImpl(Pick3PredictionService predictionService, Pick3PlaysConfig config) {
-        this.pick3PredictionService = predictionService;
-        this.config = config;
+    public Pick3PlaysServiceImpl(Pick3PredictionService pick3PredictionService) {
+        this.pick3PredictionService = pick3PredictionService;
     }
 
     @Override
     public Pick3Plays getPick3Plays(Integer winningNumber, LocalDate winningDrawDate, DrawingTime winningDrawTime,
                                     LocalDate futureDrawDate, DrawingTime futureDrawTime) {

```

Comment:
Why can't this be removed? It looks like it is no longer used in the tests.

Reply:
Remove what?  Which method or line are you talking about?

Reply:
I am guessing you are talking about the getPick3Plays() method.  I want to keep so we can use this data for logging, tracking, and statistics.